adalflow/adalflow/core/generator.py

```diff
@@ -304,24 +313,54 @@ def _extra_repr(self) -> str:
         s = f"model_kwargs={self.model_kwargs}, model_type={self.model_type}"
         return s
 
+    def _process_chunk(self, chunk: Any) -> GeneratorOutput:
```

Comment:
it specifies only one output, but you returned a tuple.

Ensure we add code linting @fm1320 by developers to check these basics

Reply:
Thanks for. the review. Yes, that's my fault. I will fix this.